### ISSUE TITLE
SSL_ERROR_SYSCALL in connection to esi.evetech.net:443

### ISSUE THREAD

**Issue by SergeyAB** [eveseat/seat#372]
Hello! I've problem with some requests:

> [2018-06-14 22:30:03] local.ERROR: cURL error 35: OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to esi.evetech.net:443  (see http://curl.haxx.se/libcurl/c/libcurl-errors.html) {"exception":"[object] (GuzzleHttp\\Exception\\ConnectException(code: 0): cURL error 35: OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to esi.evetech.net:443  (see http://curl.haxx.se/libcurl/c/libcurl-errors.html) at /opt/www/seat/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php:185)

If I try to telnet for esi.evetech.net:443, the connection establish ok:

> $ telnet esi.evetech.net 443
> Trying 34.240.24.228...
> Connected to esi.evetech.net.
> Escape character is '^]'.

> $ curl -V
> curl 7.60.0 (amd64-portbld-freebsd11.1) libcurl/7.60.0 OpenSSL/1.0.2k zlib/1.2.11 nghttp2/1.31.1
> Release-Date: 2018-05-16
> Protocols: dict file ftp ftps gopher http https imap imaps pop3 pop3s rtsp smtp smtps telnet tftp
> Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz TLS-SRP HTTP2 UnixSockets HTTPS-proxy

> $ php -v
> PHP 7.2.6 (cli) (built: May 31 2018 01:15:56) ( NTS )
> Copyright (c) 1997-2018 The PHP Group
> Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
>     with Zend OPcache v7.2.6, Copyright (c) 1999-2018, by Zend Technologies

```
+---------------+---------------+---------------+
| Package Name  | Local Version | Latest Github |
+---------------+---------------+---------------+
| Api           | 3.0.0-beta5   | 2.0.10        |
| Console       | 3.0.0-beta11  | 2.0.4         |
| Eveapi        | 3.0.0-beta23  | 2.0.14        |
| Notifications | 3.0.0-beta3   | 2.0.11        |
| Web           | 3.0.0-beta22  | 2.0.19        |
| Services      | 3.0.0-beta14  | 2.0.13        |
+---------------+---------------+---------------+
```

**Comment by warlof** [comment]
Hi,

Ensure you don't have any package update. Your openssl package is a bit old (1.1.0g on Ubuntu actually, 1.1.0f for Debian)

**Comment by SergeyAB** [comment]
I've updated packages for my OS. So... The version of OpenSSL have no so rude changes from the last. At this moment I have 2 versions of my problem:
1) Something wrong with SSL. I tried to find something and found only " try to use curl_setopt($ch, CURLOPT_SSLVERSION, 3);" but I didn't find correct file to include this option.
2) Something wrong with my network. I tried telnet to 443 port to all IP-adresses, but port is open on all IPs. I will try to find VPN to change route to evetech servers.
